src/server/api/schema.js

```diff
@@ -1455,20 +1450,23 @@ const rootMutations = {
         join_token: joinToken,
       })
       .first();
-      const INVALID_REASSIGN = () => {
-        const error = new GraphQLError("Invalid reassign request - organization not found");
-        error.code = "INVALID_REASSIGN";
-        return error;
-      };
       if (!campaign) {
-        throw INVALID_REASSIGN();
+        throw new GraphQLError("Invalid reassign request - campaign not found", {
```

Comment:
Yikes, this endpoint doesn't verify the permission.... This is exactly the way information can be leaked -- I can post do this endpoint and see how many campaigns you are running each day.  Since you're doing a PR, please fail hard with no message here.  This is a vulnerability.

Reply:
after further review I think since this is ok since the join_token is required, which is essentially access control